### PR TITLE
Use kalite executable when running commands in nstask class

### LINKED ISSUE #273: Use kalite executable when running commands in NSTASK class.
**Branch:** `feature/osx-o.16.x` (osx installer)

**Current Behaviour:** NSTASK class used `pyrun` when running ka-lite commands.

**Expected Behaviour:** NSTASK class must use `kalite` executable when running ka-lite commands.

REF: https://github.com/learningequality/installers/pull/268#discussion-diff-44636929

### PR DESCRIPTION
**Pull request by amodia**
@cpauya this fixes #273

### CODE REVIEW

**Comment by djallado** [comment]
@amodia I tested this PR using the 0.15.x build and it works.

**Comment by cpauya** [comment]
@amodia Pls set the symlink to `bin/kalite` to `/usr/local/bin/`.

File: osx/KA-Lite/KA-Lite/AppDelegate.m [hunk 1]
```diff
@@ -479,12 +433,12 @@ BOOL setLaunchAgent(NSString *source, NSString *target) {
...
 
 NSString *getUsrBinKalite() {
-    return @"/usr/bin/kalite";
+    return @"/usr/local/bin/kalite";
 }
 
 
 BOOL *checkUsrBinKalitePath() {
-    NSString *kalitePath = @"/usr/bin/kalite";
+    NSString *kalitePath = @"/usr/local/bin/kalite";
```
Comment (cpauya) [inline osx/KA-Lite/KA-Lite/AppDelegate.m]: @amodia Pls re-use `getUsrBinKalite()` here instead of hard-coding it.

**Comment by cpauya** [comment]
Yes!  We've just de-coupled the .app from Pyrun!  Thanks @amodia and @djallado.  Merging.

**Pull request merged**